game/world/objects/item.cpp

```diff
@@ -204,12 +205,19 @@ bool Item::isMulti() const {
   }
 
 bool Item::isSpellShoot() const {
+  // Whether a spell is a projectile is hardcoded in vanilla, see
+  // https://forum.worldofplayers.de/forum/threads/1460092-Stuck-in-a-charge-spell/page2?p=24786462&viewfull=1#post24786462
+  // Only hardcode Stormfist for now since other spells work with the heuristic based on target_collect_algo
   if(!isSpellOrRune())
     return false;
+  bool g1 = world.version().game==1;
+  if(g1 && spellId()==47)
```

Comment:
nitpick: better to decouple `47` in something like `const int32_t SPL_STORMFIST = 47`